### PR TITLE
CP-25712: update qemu_wrapper.py for usb pass-through with deprivileg…

### PR DESCRIPTION
**Pull request by TalonsLee**
Signed-off-by: Xin(Talons) Li <xin.li@citrix.com>

### CODE REVIEW

**Comment by lindig** [comment]
The code is fine but it needs a better commit message that explains what this patch does. As far as I can see, it introduces the `root_dir` variable to avoid using a string literals for the path in the rest of the code.

File: scripts/qemu-wrapper [hunk 1]
```diff
@@ -367,20 +367,22 @@ def main(argv):
     #qemu_args += ["-monitor", "tcp:127.0.0.1:7777,server,nowait"]
 
     if depriv:
+        root_dir = "/var/xen/qemu/{}/".format(domid)
         try:
-            os.makedirs("/var/lib/xen/root-%d/dev" % domid, 0755)
+            os.makedirs(root_dir + "dev/", 0755)
```
Comment (rosslagerwall) [inline scripts/qemu-wrapper]: No need for a trailing slash here

File: scripts/qemu-wrapper [hunk 2]
```diff
@@ -367,20 +367,22 @@ def main(argv):
     #qemu_args += ["-monitor", "tcp:127.0.0.1:7777,server,nowait"]
 
     if depriv:
+        root_dir = "/var/xen/qemu/{}/".format(domid)
```
Comment (rosslagerwall) [inline scripts/qemu-wrapper]: As Christian commented on the other PR, it would be better to not have a trailing slash here.

**Comment by rosslagerwall** [comment]
> The code is fine but it needs a better commit message that explains what this patch does. As far as I can see, it introduces the root_dir variable to avoid using a string literals for the path in the rest of the code.

The change actually renames the chroot dir to match what is in the design (`/var/xen/qemu/<domid>`). The commit message should reflect this.

**Comment by rosslagerwall** [comment]
@TalonsLee Please see the recent design update and update this PR as needed. I've tweaked it to use `/var/xen/qemu/root-<domid>` for upstream QEMU otherwise it could conflict with qemu-trad.

**Comment by TalonsLee** [comment]
@rosslagerwall 
I saw your update. But there are multiple chroot path in design.txt, 
we need to update all of them.

**Pull request merged**